src/course-home/progress-tab/grades/detailed-grades/DetailedGrades.jsx

```diff
@@ -36,11 +37,14 @@ function DetailedGrades({ intl }) {
     });
   };
 
+  const overviewTab = tabs.find(tab => tab.slug === 'outline');
+  const overviewTabUrl = overviewTab && overviewTab.url;
+
   const outlineLink = (
```

Comment:
You also need to check here that the Url exists.  Something like:

```suggestion
  const outlineLink = !overviewTabUrl ? "" : (
```

(I haven't tested the syntax.)